markdownlint-cli2-config-schema.json

```diff
@@ -0,0 +1,110 @@
+{
+	"$schema": "http://json-schema.org/draft-07/schema#",
+	"title": "markdownlint-cli2 configuration schema",
+	"type": "object",
+	"properties": {
+		"config": {
+			"$ref": "./node_modules/markdownlint/schema/markdownlint-config-schema.json"
+		},
+		"customRules": {
+			"description": "Module names or paths of custom rules to load and use when linting",
+			"type": "array",
+			"default": [],
+			"items": {
+				"description": "Module name or path of a custom rule",
+				"type": "string",
+				"minLength": 1
+			}
+		},
+		"fix": {
+			"description": "Whether to enable fixing of linting errors reported by rules that emit fix information",
+			"type": "boolean",
+			"default": false
+		},
+		"frontMatter": {
+			"description": "Regular expression used to match and ignore any front matter at the beginning of a document",
+			"type": "string",
+			"default": ""
+		},
+		"ignores": {
+			"description": "Glob expressions to ignore when linting",
+			"type": "array",
+			"default": [],
+			"items": {
+				"description": "Glob expression to ignore",
+				"type": "string",
+				"default": "**/*.md",
```

Comment:
This is the default INclude, not the default IGnore, right?

Reply:
I think there is no default for items because the default is no items?

Reply:
Makes sense. Removed in cdc30ed. I also removed the explicit `type` property from the parameter `items`. VS Code will display the tooltip with the description without a `type` specified.